articles/aks/api-server-vnet-integration.md

```diff
@@ -27,7 +27,7 @@ API Server VNet Integration is supported for public or private clusters. You can
 ## Limited availability
 
 > [!IMPORTANT]
-> **API Server VNet Integration has limited availability and capacity in certain regions.**  
+> **API Server VNet Integration is supported in all regions but may have limited capacity in certain regions.**  
```

Comment:
this is not true. There are limited capacity in some regions, so it is not GA in all regions.